compat/apester-rules-configuration.json

```diff
@@ -0,0 +1,21 @@
+{
+  "rules": [{
+    "class": "IgnoreRule",
+    "selector": "//p/script[contains(@src,'apester')]"
```

Comment:
Thanks @diegoquinteiro, you're correct. I've changed the file properly. CI is running checks...